rotkehlchen/chain/bitcoin/manager.py

```diff
@@ -451,18 +495,24 @@ def decode_op_return(self, tx: BitcoinTx, raw_script: str) -> HistoryEvent | Non
             notes=notes,
         )
 
-    def _decode_single_input_transfers(
+    def _decode_single_input_or_output_transfers(
```

Comment:
this method is misleading. it reads like either an input/output gets decoded instead of 1-input & multiple outputs.